src/pages/getting-started/app-guides/helpdesk-widgets/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+weight: 80
+category: "getting-started"
+subcategory: "app-guides"
+title: "HelpDesk Widgets"
+tagline: "HelpDesk Widgets are web apps loaded inside the HelpDesk Application."
+desc: "HelpDesk Widgets are web apps loaded inside the HelpDesk Application. You can use them to embed and display static content, embed your SaaS service, or query external services."
+---
+
+## Introduction
+
+App Widgets are web apps loaded inside the HelpDesk Application. They live in iframes and can interact and communicate with the HelpDesk App. You can use them to:
+
+- embed and display static content, for example, knowledge base articles, conversation prompts, or context information.
```

Comment:
Most of these use cases are more LiveChat-specific. Since HelpDesk is a ticketing system, I don't think that creating a store with in-app purchases is applicable here. Knowing what HelpDesk does, try to think about the types of applications developers might build. I'll also send you some inspiration via DM